ResourceType.dhall

```diff
@@ -0,0 +1,50 @@
+let ResourceType
+    : ∀(_params : { Source : Type, SourceGetParams : Type }) →
+        { Type : Type
+        , default :
+            { privileged : Optional Bool
+            , params : Optional _params.SourceGetParams
+            , check_every : Optional Text
+            , tags : Optional (List Text)
+            , unique_version_history : Optional Bool
+            }
+        }
+    = λ(_params : { Source : Type, SourceGetParams : Type }) →
```

Comment:
Why not just getting the `\(resource-types : Tyoe) ->` as anyways `resource-types/*.dhall` holds all the required input.